src/SonarScanner.MSBuild.Tasks/GetAnalyzerSettings.cs

```diff
@@ -192,27 +192,14 @@ bool IsTestAnalysisSupported()
 
             // See https://github.com/SonarSource/sonar-scanner-msbuild/issues/561
             // Legacy behaviour is to overwrite.
-            // The new (SQ 7.4+) behaviour is to merge only if sonar.[LANGUAGE].roslyn.ignoreIssues is false.
             var serverVersion = config?.FindServerVersion();
             if (serverVersion == null || serverVersion < new Version("7.4"))
```

Comment:
given it's now only an if statement, can you make the method more positive-logic oriented?

```
if (serverVersion != null && serverVersion >= new Version("7.4"))
{
  return true;
}
else
{
               logger.LogInfo(Resources.AnalyzerSettings_ExternalIssueNotSupported, SonarProduct.GetSonarProductToLog(config?.SonarQubeHostUrl));
                return false;
}
```

Like this, the answer to `ShouldMergeAnalysisSettings` is straightforward